tests/lora/utils.py

```diff
@@ -85,8 +85,13 @@ class PeftLoraLoaderMixinTests:
     unet_kwargs = None
     transformer_cls = None
     transformer_kwargs = None
+    vae_cls = AutoencoderKL
     vae_kwargs = None
 
+    text_encoder_target_modules = ["q_proj", "k_proj", "v_proj", "out_proj"]
```

Comment:
The tiny-random-t5 models has layers named as `q`, `k`, `v` and `o`. We set this correctly in the Cog test